### ISSUE TITLE
Fix Android tokens output in 10.1.0

### ISSUE THREAD

**Issue by danoc** [thumbtack/thumbprint-tokens#11]
Getting this error in the JitPack build:
```
* What went wrong:
Execution failed for task ':packageDebugResources'.
> /home/jitpack/build/dist/android/index.xml: Error: Can't determine type for tag '<item name="tp_duration_1">75</item>'
```

**Comment by danoc** [comment]
I think the issue here is that it shouldn't be within the `<item>` tag.

[Based on the examples in the Android docs](https://developer.android.com/guide/topics/resources/more-resources#Integer), I think that `Integer` is the correct type.

**Comment by danoc** [comment]
Actually it looks like it should be double:
https://github.com/amzn/style-dictionary/blob/master/examples/advanced/custom-formats-with-templates/templates/android-xml.template

**Comment by danoc** [comment]
This didn't seem to fix it. Still getting errors on JitPack:

```

> Task :packageDebugResources FAILED
/home/jitpack/build/dist/android/index.xml: Error: Can't determine type for tag '<double name="tp_duration_1">75</double>'

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':packageDebugResources'.
> /home/jitpack/build/dist/android/index.xml: Error: Can't determine type for tag '<double name="tp_duration_1">75</double>'
```

**Comment by danoc** [comment]
This was fixed in `10.1.3`.
